components/ContractSummary.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card class="mx-auto my-12" max-width="500">
+    <v-card-title
+      ><a :href="etherscanLinkContract">{{
+        this.contractName
+      }}</a></v-card-title
+    >
+    <v-card-subtitle>{{ this.contractType }}</v-card-subtitle>
+    <v-card-text>
+      <v-list-item>
+        <v-list-item-title> Expiration Time </v-list-item-title>
+        <v-list-item-subtitle>{{ this.expirationTime }}</v-list-item-subtitle>
+      </v-list-item>
+
+      <v-list-item>
+        <v-list-item-title> Collateral </v-list-item-title>
+        <v-list-item-subtitle
+          ><a :href="etherscanLinkCollateral">{{
+            this.contractDetails.collateralToken
+          }}</a></v-list-item-subtitle
+        >
+      </v-list-item>
+
+      <v-list-item>
+        <v-list-item-title> Collateral Per Pair</v-list-item-title>
+        <v-list-item-subtitle>{{
+          this.contractDetails.collateralPerPair
+        }}</v-list-item-subtitle>
+      </v-list-item>
+
+      <v-list-item>
+        <v-list-item-title> Price Feed </v-list-item-title>
+        <v-list-item-subtitle
+          ><a :href="priceFeedLink">{{
+            this.contractDetails.priceIdentifier
+          }}</a></v-list-item-subtitle
+        >
+      </v-list-item>
+
+      <v-list-item
+        v-for="(parameterInfo, index) in libraryConfiguration"
+        :key="index"
+      >
+        <v-list-item-title>{{ parameterInfo.name }}</v-list-item-title>
+        <v-list-item-subtitle>{{ parameterInfo.value }}</v-list-item-subtitle>
+      </v-list-item>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from "nuxt-property-decorator"
+import { LSPConfiguration } from "~/types"
+
+const addresses: Record<string, string> = require("~/addresses.json")
```

Comment:
Mind if we import instead? The biggest gain is on import being async vs a require statement here being synchronous.

Reply:
I haven't found a way to define the typescript type of json object on import. Therefore I would like to keep it at require right now.

Reply:
Also I get all kinds of errors when I use import instead. Sucha s "cant find module abis"
I would like to stick to require for now to not waste too much time on this, but of course feel free to adjust it to import at a later point.